python/ray/node.py

```diff
@@ -394,16 +394,19 @@ def _make_inc_temp(self, suffix="", prefix="", directory_name=None):
         raise FileExistsError(errno.EEXIST,
                               "No usable temporary filename found")
 
-    def get_log_file_names(self, name, unique=False):
-        """Generate partially randomized filenames for log files.
+    def get_log_file_handlers(self, name, unique=False):
```

Comment:
I might call this `get_log_file_handles` though, unless "file handler" is an existing term I'm not familiar with

Reply:
Ha, good point, I'm not sure how that `r` snuck in there.